x-pack/plugins/security_solution/scripts/run_playwright/start_playwright.js

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
Same here with the above mentioned.

Reply:
https://github.com/elastic/kibana/pull/190803#discussion_r1738332169